lib/experiment/remote/client.rb

```diff
@@ -102,9 +102,7 @@ def do_fetch(user, timeout_millis)
       http = PersistentHttpClient.get(@uri, { read_timeout: read_timeout }, @api_key)
       request = Net::HTTP::Post.new(@uri, headers)
       request.body = user_context.to_json
-      if request.body.length > 8000
```

Comment:
why'd you remove this? Now we're logging the warning on every request, regardless of user size.

Reply:
This was rubocop reformatting the file by including the `if` statement at the end, the logic still exists in the new line 105